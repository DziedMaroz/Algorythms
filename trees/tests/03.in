1
2
3
4
15
9
8
16
17
18
19
