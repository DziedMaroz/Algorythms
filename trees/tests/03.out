1
2
3
4
16
9
8
17
18
19
